### ISSUE TITLE
CI for the repository

### ISSUE THREAD

**Issue by jasmainak** [cjayb/hnn-core-examples#1]
I was wondering if we could host some longer/educational examples on this repository that may not run on the main `hnn-core` repository. We could even have a CI but instead of having it run all the examples, we could set up a [cron job](https://support.circleci.com/hc/en-us/articles/115015481128-Scheduling-jobs-cron-for-builds-) where each time it runs a different example (e.g., based on the day or time of the day). So, we'd know if something broke but it wouldn't necessarily be blockers for PRs in hnn-core.

**Comment by cjayb** [comment]
Well I definitely like that idea! But wonder if it wouldn't make sense to have it in the jonescompneurolab-organization? I guess the argument against moving it there would be curation... Using personal accounts (like mine) would allow a more liberal playground, with a big fat disclaimer of not being a part of the "official" HNN-realm. Is that what you're thinking too?

**Comment by jasmainak** [comment]
yeah, I am not opposed to moving it under jonescompneurolab :) But perhaps once there are a few examples brewing? Certainly having it under a personal account gives a more liberal playground. HNN-core was hosted under jasmainak/mne-neuron for more than a year and I enjoyed the freedom and dictatorship ;-) Anyhow it's up to you!

**Comment by cjayb** [comment]
OK, let's keep them here for a while longer then! I'll write a README to make it clear this is not an "official" branch (yet). When we have more examples and know how this will develop, we can move it to the organisation and rename it.

I should set up CI in this repo, right? Not have `hnn-core` trigger anything, but run CircleCI on cron?
